sql/binding_contexts.h

```diff
@@ -350,7 +350,7 @@ struct SqlExpressionExtractScope: public SqlBindingScope {
 
     /// Input variables, for when they are known.  Will be null
     /// when the input is unknown.
-    std::shared_ptr<RowValueInfo> inputInfo;
```

Comment:
Because it can be an embedding now too. We can put it back after we make EmbeddingValueInfo derive from RowValueInfo.